lte/gateway/c/core/oai/tasks/amf/amf_authentication.cpp

```diff
@@ -914,6 +923,7 @@ int amf_send_authentication_request(amf_context_t* amf_ctx,
 // Fetch the serving network name
 static int calculate_amf_serving_network_name(amf_context_t* amf_ctx,
                                               unsigned char* snni) {
+  OAILOG_FUNC_IN(LOG_AMF_APP);
```

Comment:
While returning, right  module is not included

Reply:
Comment addressed.